docs/src/terminology.md

```diff
@@ -110,13 +110,17 @@ The first [block](terminology.md#block) in the chain.
 
 The configuration file that prepares the [ledger](terminology.md#ledger) for the [genesis block](terminology.md#genesis-block).
 
+## gossip 
+
+A protocol for peer-to-peer communication. Validators utilize a gossip service to ensure that information is available to all other nodes in a cluster. 
+
 ## hash
 
 A digital fingerprint of a sequence of bytes.
 
 ## inflation
 
-An increase in token supply over time used to fund rewards for validation and to fund continued development of Solana.
+An increase in the token supply, used to fund validator rewards and Solana protocol development. Validator rewards trickle down to stake accounts that delegate their stake weight.
```

Comment:
I like the changes to the first sentence.  The second sentence is awkward: trickle-down makes me think of economics, and one delegates stake, not stake weight. Might want to just link to the docs section that describes Solana inflation instead.